src/atoms/affine/index.jl

```diff
@@ -78,12 +78,12 @@ end
 
 ## API Definition begins
 
-getindex{T <: Real}(x::AbstractExpr, rows::AbstractArray{T, 1}, cols::AbstractArray{T, 1}) = IndexAtom(x, rows, cols)
-getindex{T <: Real}(x::AbstractExpr, inds::AbstractArray{T, 1}) = IndexAtom(x, inds)
+getindex(x::AbstractExpr, rows::AbstractArray{<:Real, 1}, cols::AbstractArray{<:Real, 1}) = IndexAtom(x, rows, cols)
```

Comment:
`AbstractArray{_, 1}` can be reformatted as `AbstractVector{_}`

Reply:
Shall I change `AbstractArray{_, 2}` to `AbstractMatrix{_}` too?

Reply:
Yes